src/Text/Pandoc/Parsing.hs

```diff
@@ -61,13 +61,19 @@ module Text.Pandoc.Parsing ( module Text.Pandoc.Sources,
                              lineBlockLines,
                              tableWith,
                              tableWith',
+                             tableWithSpans,
+                             tableWithSpans',
                              widthsFromIndices,
                              gridTableWith,
                              gridTableWith',
                              TableComponents (..),
                              TableNormalization (..),
                              toTableComponents,
                              toTableComponents',
+                             toTableComponentsWithSpans,
+                             toTableComponentsWithSpans',
+                             singleRowSpans,
+                             singleColumnSpans,
```

Comment:
This looks great! Just be sure to use the string `[API change]` in your commit message and indicate what has changed (this is an exported module).